### DISCUSSION TITLE
How to do something when initializing the game?

### DISCUSSION THREAD

**Discussion by S-N-O-R-L-A-X** [boardgameio/boardgame.io#1201] [Q&A]
I am really new to this framework. How can I initialize the game? BTW, gitter is 404 to me. Is there another platform to discuss?
For example, if I want to make both players have draw some cards from deck after importing, what can I do?

```ts
const importAllCards = (): Card[] => {
	return cards as Card[];
}

export const MyGame: Game<MyGameState> = {
	setup: () => ({
		deck: importAllCards(),
		Hands: [[], []]
	}),
	turn: {
		stages: {
			draw: {
				moves: { DrawCard },
				next: "play",
			},

			play: {
				moves: { PlayCard },
			},
		}
	},
	moves: {
		shuffle({ G, random }) {
			if (G.deck.length === 0) {
				G.deck = importAllCards();
			}
			G.deck = random.Shuffle(G.deck);
		},
	},
};
```

**Comment by fladrif** [comment]
Do you mean you want the players to start the game with some cards in hand? If so you can do that in the setup function. The return is a game object that you can update before returning, so you can import the deck and pop cards off into the player's hands first.
